src/region/RegionContainer.tsx

```diff
@@ -36,11 +37,16 @@ export type Props = {
 
 export const mapStateToProps = (state: AppState, { location }: { location: number }): Props => {
     const staged = getCreatorStagedForLocation(state, location);
+    const isDiscoverabilityFeatureEnabled = getIsDiscoverabilityFeatureEnabled(state);
+
+    const isCreating = isDiscoverabilityFeatureEnabled
+        ? getAnnotationMode(state) === Mode.REGION || getAnnotationMode(state) === Mode.NONE
+        : getAnnotationMode(state) === Mode.REGION;
```

Comment:
Could we make this change (to have create region annotations enabled by default) in a separate PR?

Reply:
Could you describe why it would be better to move it out to a separate PR?

Reply:
Typically we like to keep a commit to one change, for change history clarity and in case we need to revert, and it seems like the scope of this PR is to `put region layer under text layer`. This other change, `make create region available by default` doesn't seem tied to this change and could be separate. Maybe also bundle it with the change to apply `.ba-is-create--region` since it's related

Reply:
So to clarify, all FF related changes in this PR should be moved out to a separate PR? Not just lines 40-44 in RegionContainer.tsx.

Reply:
I think you still need the FF changes in this PR to only layer it below when it is discoverability -- but the changes to support creating regions by default should move out